src/Tools/dotnet-monitor/ParameterCapturing/CaptureParametersOperation.cs

```diff
@@ -33,13 +40,21 @@ internal sealed class CaptureParametersOperation : IInProcessOperation
 
         public Task Started => _capturingStartedCompletionSource.Task;
 
-        public CaptureParametersOperation(IEndpointInfo endpointInfo, ProfilerChannel profilerChannel, ILogger logger, CaptureParametersConfiguration configuration, TimeSpan duration)
+        public CaptureParametersOperation(
+            IEndpointInfo endpointInfo,
+            ProfilerChannel profilerChannel,
+            ILogger logger,
+            CaptureParametersConfiguration configuration,
+            TimeSpan duration,
+            CapturedParameterFormat format)
         {
             _profilerChannel = profilerChannel;
             _endpointInfo = endpointInfo;
             _logger = logger;
             _configuration = configuration;
             _duration = duration;
+            _parameterCapturingStore = endpointInfo.ServiceProvider.GetRequiredService<IParameterCapturingStore>();
```

Comment:
nit: Let's retrieve the store via the `CaptureParametersOperationFactory` constructor DI instead of reaching into the service provider here to be consistent with how we do DI elsewhere